index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <div class="container">
+      <h1 id="title">Microverse Survey Form</h1>
+      <p id="description">Thank you for taking the time to fill this form!</p>
+
+      <form id="survey-form">
+        <div>
+          <label id="name-label">Full name</label>
+          <input
+            type="text"
+            id="name"
+            placeholder="Write your name here"
+            required
+          />
+        </div>
+        <div>
+          <label id="email-label">Email</label>
+          <input
+            type="email"
+            id="email"
+            placeholder="Enter Your Email"
+            required
+          />
+        </div>
+        <div>
+          <label id="number-label">Number</label>
+          <input
+            type="number"
+            id="number"
+            placeholder="Enter Your Number"
+            min="0"
+            max="1000"
+          />
+        </div>
+
+        <div>
+          <div>
+            <label id="name-label">Name</label>
+            <input
+              type="text"
+              id="name"
+              placeholder="Write your name here"
+              required
+            />
+          </div>
+          <div>
+            <label id="email-label">Email</label>
+            <input
+              type="email"
+              id="email"
+              placeholder="Enter Your Email"
+              required
+            />
+          </div>
+          <div>
+            <label id="number-label">Number</label>
+            <input
+              type="number"
+              id="number"
+              placeholder="Enter Your Number"
+              min="0"
+              max="1000"
+            />
+          </div>
+        </div>
+
+        <select id="dropdown">
+          <option>Option1</option>
+          <option>Option2</option>
+        </select>
+        <div>
+          <input value="radio" name="Radio" type="radio" />
+          <label>Option1</label>
+          <input value="radio" name="Radio" type="radio" />
+          <label>Option2</label>
+        </div>
+
+        <div>
```

Comment:
kindly follow the HTML and CSS best practices

Reply:
[OPTIONAL] Well done using inline comments 😉 but make sure to address this differently, the student must understand first what part of the code can be improved and what exactly it will bring to his project. You added an inline comment on a closing `div` and an opening one, providing more context will help the student implement the changes better. 💯 It could be something like:
```
Well done on this section but it could be even better if you used HTML and CSS best practices, this would make your code more readable and your website would be more indexable by browsers. To learn more about HTML and CSS best practices. See here (provide link)
```
Keep in mind that this is just an example and I hope it helps. 🤓